src/frontend/Komponenter/Personoversikt/Behandlingsoversikt.tsx

```diff
@@ -68,21 +60,19 @@ const lagTilbakekrevingslenke = (eksternFagsakId: number, behandlingId: string):
     return `${tilbakekrevingBaseUrl()}/fagsystem/EF/fagsak/${eksternFagsakId}/behandling/${behandlingId}`;
 };
 
-const Behandlingsoversikt: React.FC<{ fagsakId: string }> = ({ fagsakId }) => {
-    const [fagsak, settFagsak] = useState<Ressurs<Fagsak>>(byggTomRessurs());
-    const [tekniskOpphørFeilet, settTekniskOpphørFeilet] = useState<boolean>(false);
-    const [kanStarteRevurdering, settKanStarteRevurdering] = useState<boolean>(false);
-
-    const [visLagBehandlingModal, settVisLagBehandlingModal] = useState<boolean>(false);
+const Behandlingsoversikt: React.FC<{ fagsakPerson: IFagsakPerson }> = ({ fagsakPerson }) => {
```

Comment:
Denne tenker jeg att man hadde kunnet splittet opp som dere gjort, men att i stedet for å ha en komponent som heter `FagsakOvergangsstønad` og `FagsakBarnetilsyn`, å gjennbruke den samme "Fagsaken" med en param for tittel som skal vises for hver fagsak. 
Vi trenger sannsynligvis både tilbakekreving og teknisk opphør for begge disse?

Reply:
Det gjør vi nok, men ser ikke den store gevinsten av å generalisere på det nivået helt ennå. Synes det er fint å se at "Barnetilsyn" er plain, mens OS har litt mer - så har det vel vært diskutert fra Lars' side at denne siden nok må bli litt "annereledes" på et eller annet tidspunkt uansett når vi får flere stønader?

Reply:
På sikt kunne man kanskje brukt en felleskomponent, men jeg synes kanskje det blir enklere om vi har de tre eksplisitte komponentene for hver av stønadtypene. Mtp. på featuretoggling av knapper for revurdering, tilbakekrevingsbehandlinger osv.

Reply:
Jeg tenkte det hadde vært enklere(og mindre arbeid) hvis man bare hadde haft en fagsakvisning, då både overgangsstønad og barnetilsyn er veldig like. 
Det er ikke stor generalisering det er snakk om, det er i prinsipp det dere gjort for FagsakOvergangsstønad, men att den hadde blitt gjennbrukt av Barnetilsyn.

Når vi kan opprette en behandling for barnetilsyn, så kan vi vel i prinsipp opprette en revurdering og? 
Og tilbakekreving er vel allerede klart for å håndtere barnetilsyn og? 

Det som er behov for å featuretoggle er vel barnetilsyn eller ikke barnetilsyn? :P

Reply:
Tror ikke at vi åpner for alle behandlings samtidig ved barnetilsyn? Jeg ser for meg at vi starter med førstegangsbehandling og gradvis dekker flere sakstyper